x-pack/plugins/security_solution/public/timelines/components/edit_data_provider/index.tsx

```diff
@@ -181,102 +180,99 @@ export const StatefulEditDataProvider = React.memo<Props>(
 
     useEffect(() => {
       disableScrolling();
-      focusInput();
       return () => {
         enableScrolling();
       };
     }, []);
 
     return (
       <EuiPanel paddingSize="s">
-        <EuiFocusTrap data-test-subj="focusTrap">
-          <EuiFlexGroup direction="column" gutterSize="none">
-            <EuiFlexItem grow={false}>
-              <EuiFlexGroup gutterSize="s" direction="row" justifyContent="spaceBetween">
-                <EuiFlexItem grow={false}>
-                  <EuiFormRow label={i18n.FIELD}>
-                    <EuiComboBox
-                      data-test-subj="field"
-                      isClearable={false}
-                      onChange={onFieldSelected}
-                      options={getCategorizedFieldNames(browserFields)}
-                      placeholder={i18n.FIELD_PLACEHOLDER}
-                      selectedOptions={updatedField}
-                      singleSelection={{ asPlainText: true }}
-                      style={{ width: `${FIELD_COMBO_BOX_WIDTH}px` }}
-                    />
-                  </EuiFormRow>
-                </EuiFlexItem>
-
-                <EuiFlexItem grow={false}>
-                  <EuiFormRow label={i18n.OPERATOR}>
-                    <EuiComboBox
-                      data-test-subj="operator"
-                      isClearable={false}
-                      onChange={onOperatorSelected}
-                      options={operatorLabels}
-                      placeholder={i18n.SELECT_AN_OPERATOR}
-                      selectedOptions={updatedOperator}
-                      singleSelection={{ asPlainText: true }}
-                      style={{ width: `${OPERATOR_COMBO_BOX_WIDTH}px` }}
-                    />
-                  </EuiFormRow>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiFlexItem>
-
-            <EuiFlexItem grow={false}>
-              <EuiSpacer size="m" />
-            </EuiFlexItem>
+        <EuiFlexGroup direction="column" gutterSize="none">
```

Comment:
Only change here was removing the focus trap right?

Reply:
I also added an autofocus to the Field component as without it the page experienced a jitter in the Field menu on render.